core/internal/corelib/summarylib.go

```diff
@@ -18,8 +19,13 @@ func JsonifyItems[V genericItem](items []V) (string, error) {
 	jsonMap := make(map[string]interface{})
 
 	for _, item := range items {
-		var value interface{}
-		if err := json.Unmarshal([]byte(item.GetValueJson()), &value); err != nil {
+		var (
+			value interface{}
+			err   error
+		)
+		// TODO: use simplejsonext for now until we replace the usage of json with protocol buffer
+		// and proto json marshaler
```

Comment:
super nit: I think this comment could be removed, or at least moved to above the import. The code below it doesn't look like it's using something other than the json library since the import is called `json`